ansible/roles/debops.gunicorn/templates/etc/systemd/system/gunicorn@.service.j2

```diff
@@ -13,7 +13,7 @@ Wants=network-online.target
 PermissionsStartOnly=true
 EnvironmentFile=-/etc/default/gunicorn-%i
 PIDFile=/run/gunicorn-%i/%i.pid
-ExecStart=/usr/bin/gunicorn --config=/etc/gunicorn/%i.conf.py ${GUNICORN_OPTS}
+ExecStart=/usr/bin/{{ item.binary if (item.binary is defined) else gunicorn__binary }} --config=/etc/gunicorn/%i.conf.py ${GUNICORN_OPTS}
```

Comment:
This change doesn't make sense - per-item variables are available in the override file, not in the main service file. You should only specify `gunicorn__binary` here.